schema/mappings/gold-to-mixs.sssom.tsv

```diff
@@ -40,8 +43,8 @@ gold.vocab:soil_store_condition	soil_store_condition	skos:exactMatch	mixs:store_
 gold.vocab:soil_link_climate_info	soil_link_climate_info	skos:exactMatch	mixs:link_climate_info	link_climate_info	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
 gold.vocab:soil_annual_season_temp	soil_annual_season_temp	skos:closeMatch	mixs:annual_temp	annual_temp	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
 gold.vocab:soil_annual_season_temp	soil_annual_season_temp	skos:closeMatch	mixs:season_temp	season_temp	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
-gold.vocab:soil_annual_season_precpt	soil_annual_season_precpt 	skos:closeMatch	mixs:season_precpt	season_precpt	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
-gold.vocab:soil_annual_season_precpt	soil_annual_season_precpt 	skos:closeMatch	mixs:annual_precpt	annual_precpt	SSSOMC:HumanCurated	gold.vocab	mixs	1	.
```

Comment:
why did you drop this one? it's a closeMatch so ok if not 1:1

Reply:
I don't remember dropping these, but I must have done it ... I will add the mappings back. Right now the algorithm only looks exact matches (i.e., the gold fields are only mapped to exact matches).